pkg/androidSdk/complete.go

```diff
@@ -0,0 +1,61 @@
+package androidSdk
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+	api "github.com/aerogear/android-sdk-operator-poc/pkg/apis/androidsdk/v1"
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// deletePod cleans up suceeded pods and sets the resource's status to "Done"
+// once complete
+func deletePod(h *Handler, helper *api.AndroidSDK, podName string) error {
```

Comment:
need to change `helper` to a better name, but didn't want to collide with `sdk`. 
open to suggestions :)